cdhweb/static_src/global/layout/streamfields.scss

```diff
@@ -1,6 +1,21 @@
 .streamfields-wrapper {
-  //   TODO adjust this when we add full-bleed bg components
   & > * + * {
-    margin-block-start: var(--streamfield-space);
+    // Default big space above each streamfield
+    margin-block-start: var(--streamfield-space-lg);
+
+    // Cases where there should be a small space above:
+    &:where(:is(.block-image, .block-table, .block-note)) {
+      margin-block-start: var(--streamfield-space-sm);
+    }
+    &:where(.block-heading + .block-rich_text) {
+      margin-block-start: var(--streamfield-space-sm);
+    }
+  }
+
+  // Outdented components:
```

Comment:
There will be more outdented components in the future. For now it's just this h2 component, which looks like this:
<img width="851" alt="Screenshot 2024-05-23 at 2 23 16 PM" src="https://github.com/springload/cdh-web/assets/1134713/80a918a6-d69f-42bd-a319-63bba573caf7">

Reply:
Have added code to do this to the migrated content also.